mapdata/WarcraftLegacies/AbilityData/Skin/826814529.json

```
@@ -75,7 +75,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Restores 100 hit points to friendly units in a very small radius."
+      "Value": "Restores 200 hit points to friendly units in a 250 radius."
```

```suggestion
      "Value": "Restores 200 hit points to friendly units within a range of 250."
```
Just keeping it standard.